integrations.json

```diff
@@ -1624,6 +1624,34 @@
       }
     ]
   },
+  {
+    "name": "Log4Net",
+    "method_replacements": [
+      {
+        "caller": {},
+        "target": {
+          "assembly": "log4net",
+          "type": "log4net.Util.AppenderAttachedImpl",
+          "method": "AppendLoopOnAppenders",
+          "signature_types": [
+            "System.Int32",
+            "log4net.Core.LoggingEvent"
+          ],
+          "minimum_major": 2,
+          "minimum_minor": 0,
+          "minimum_patch": 0,
+          "maximum_major": 2,
+          "maximum_minor": 65535,
+          "maximum_patch": 65535
+        },
+        "wrapper": {
+          "assembly": "Datadog.Trace, Version=1.28.3.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
```

Comment:
Don't forget to rebase+regenerate before merge 🙂

Reply:
Done 👍🏼